source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,170 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collection using **read operations**.
+
+Read operations allow you to do the following: 
+
+- Retrieve a subset of documents from your collection by using a :ref:`find operation <retrieve-find>` 
+- Perform transformations on retrieved documents from your collection using the :ref:`aggregate operation <retrieve-aggregate>`. 
+
+Sample Data
+~~~~~~~~~~~
+
+To follow along with the examples on this page, run the following
+snippet to load the documents into the ``ratings`` collection in the
+``tea`` database:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/retrieve.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+Each document contains the tea a person tried and their rating on a
+scale of 10, which corresponds to the ``name`` and ``rating`` fields.
+
+.. note::
+
+   Each example truncates the ``ObjectID`` value since the driver
+   generates them uniquely. 
+
+.. _retrieve-find:
+
+Find operations
+---------------
+
+Use a **find operation** to retrieve a subset of your existing data.
```

Comment:
nit: I think this description isn't accurate and is redundant with the information presented below. It also doesn't match the plurality of the next sentence, which is more accurate.

```suggestion
Use **find operations** to retrieve data from MongoDB.
```